src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Figure['shape'] = 'triangle';
+
+  public a: number;
+
+  public b: number;
+
+  public c: number;
```

Comment:
```suggestion
  public shape: Figure['shape'] = 'triangle';
  public a: number;
  public b: number;
  public c: number;
```
There is no need to add empty lines all the time